mods/cnc/rules/defaults.yaml

```diff
@@ -122,9 +122,12 @@
 		CloakDelay: 90
 		CloakSound: trans1.aud
 		UncloakSound: trans1.aud
-		RequiresCondition: cloak
+		RequiresCondition: cloak && !uncloak
 	ExternalConditions@CLOAK:
 		Conditions: cloak
+	GrantConditionOnDamageState@UNCLOAK:
+		Condition: uncloak
```

Comment:
I would really like it if we could use more descriptive condition names.  Can you please rename these to `cloak-force-disabled`?